packages/types/src/index.ts

```diff
@@ -16,7 +16,8 @@ export { default as Balance } from './Balance';
 // export * from './Bft;
 export { default as Block } from './Block';
 export { default as BlockNumber } from './BlockNumber';
-export { default as Bool } from './Bool';
+// NOTE We are exporting Bool as bool to match with Rust
```

Comment:
Do we need to follow Rust types' case?

These are JS classes after all, imo we should follow JS guidelines above Rust code-similarity.